### PR TITLE
sync en zh yaml in content/zh/example/admin  direcroty 

### PR DESCRIPTION
**Pull request by tanjunchen**
/cc @tengqm 
/cc @xichengliudui 
ref  https://github.com/kubernetes/website/issues/19765

### CODE REVIEW

File: content/zh/examples/admin/dns/busybox.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-spec:
-  containers:
-  - name: busybox
-    image: busybox:1.28
-    command:
-      - sleep
-      - "3600"
-    imagePullPolicy: IfNotPresent
-  restartPolicy: Always
+apiVersion: v1
```
Comment (tengqm) [inline content/zh/examples/admin/dns/busybox.yaml]: can you articulate what have been changed?

Comment (tanjunchen) [reply content/zh/examples/admin/dns/busybox.yaml]: Sorry, wrong. revert it

Comment (tanjunchen) [reply content/zh/examples/admin/dns/busybox.yaml]: Problems with delimiters in windows and unix

**Comment by tanjunchen** [comment]
/cc @tengqm 
done

File: content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml [hunk 2]
```diff
@@ -9,8 +9,8 @@ spec:
     resources:
       limits:
         memory: "1Gi"
-        cpu: "800m"
+        cpu: "800m"      
       requests:
         memory: "700Mi"
         cpu: "400m"
-
+
```
Comment (tengqm) [inline content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml]: Extra spaces added. pls remove.

File: content/zh/examples/admin/resource/quota-mem-cpu-pod.yaml [hunk 3]
```diff
@@ -9,8 +9,8 @@ spec:
     resources:
       limits:
         memory: "800Mi"
-        cpu: "800m"
+        cpu: "800m" 
       requests:
         memory: "600Mi"
         cpu: "400m"
-
+
```
Comment (tengqm) [inline content/zh/examples/admin/resource/quota-mem-cpu-pod.yaml]: Extra spaces added. pls remove.

File: content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml (hunk 2 above)
Comment (tanjunchen) [reply content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml]: The original English yaml has extra spaces, so these two files are not updated.
I detected it by using a script
done

**Comment by tengqm** [comment]
@tanjunchen Can you try modify the PR a little bit and push a new commit? Looks netlify is not working even after I tried close and reopen this one.

**Comment by tanjunchen** [comment]
> @tanjunchen Can you try modify the PR a little bit and push a new commit? Looks netlify is not working even after I tried close and reopen this one.

i see. it fails due to the code is not updated due to https://github.com/kubernetes/website/pull/19764

**Comment by tanjunchen** [comment]
CI should report code conflicts, not system errors.

**Pull request merged**